apps/editor/src/app/atoms/dashboard/subscriberChart.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {useMemo} from 'react'
+import {useTranslation} from 'react-i18next'
+import {BarChart, Bar, XAxis, YAxis, ResponsiveContainer} from 'recharts'
```

Comment:
Love recharts <3

Reply:
I think you mention you just added it as well?

Reply:
Yeah :D